macbuild/ReadMe.md

```diff
@@ -84,4 +84,25 @@ $ ./makeDMG4mac.py -p qt5.pkg.macos-HighSierra-release -m
 ```
 
 By: Kazzz (January 16, 2018)
+
+# 5. Alternative building options
+### 5.1 Python 3.6 from brew, Qt 5.9.4 from offline installer
+
+Homebrew's installation of python3 places a `Python.framework` in `/usr/local/opt/python/Frameworks/Python.framework/`, which you can use to build KLayout from. Qt can be downloaded for [offline installation](https://www1.qt.io/offline-installers/). You can place it in your home folder: e.g. `/home/username/Qt5.9.4/`. Given these two dependencies, you can successfully compile KLayout with the following commands:
+
+```
+# Build step
+./build4mac.py -p B36 -q Qt5Custom
+
+# Deploy step
+./build4mac.py -p B36 -q Qt5Custom -y  # normal deploy
+./build4mac.py -p B36 -q Qt5Custom -y -v 3 2>&1 | tee qt5.pkg.macos-HighSierra-release.log  # deploy with debug options
+
+# Packaging step
+./makeDMG4mac.py -p qt5.pkg.macos-HighSierra-release -m -q Qt594
+
+```
+
+PS: If you get a syntax error in one of ruby's libraries because it is not compatible with C++11, do not fret. You only have to change one offending file.
```

Comment:
This works but it requires editing system files. On mac, not even root can do this unless system integrity protection is disabled.

Reply:
Instead you can ignore that type of warning with a flag in CXXARGS.
Build/Deploy tested on HighSierra.

Reply:
Thanks. Maybe I should remove this comment from the readme.